main.js

```diff
@@ -2871,9 +2867,9 @@ sCB.addEventListener( 'click', function( e ) {
     motionBlurFrames: ( 960 / framerate ) * ( document.querySelector('input[name="motion-blur"]').checked ? 1 : 0 ),
     quality: 100,
     format: document.querySelector('input[name="encoder"]:checked').value,
-    workersPath: './ccapture_workers/',
-    timeLimit: 60,  // This is just to help prevent the feature from accidentally filling up the hard drve
-    frameLimit: 1200,
+    workersPath: './components/CCapture/',
```

Comment:
not sure this is actually used, but I updated it anyway for now
I left timeLimit and frameLimit in there in case you want them for anything, but they're not being used at the moment.

Reply:
I think the pngs are the most important format to have working. Going straight to mpg is a convienience, but up until now I've been encoding from pngs with ffmpeg. I'll merge, sync, and try out your fixes now.